README.md

```diff
@@ -7,7 +7,16 @@
 [![NPM](https://nodei.co/npm/simpledatevalidator.png)](https://www.npmjs.com/package/simpledatevalidator/)
 
 ## How to use?
-Pass date and date format to isValid function. Function will validate and return true/false;
+Pass date and date format to `isValidDate` function.
+Function will validate and return true or false.
+
+```
+// Create Instance of the DateValidator
+var DateValidator = require('./index');
```

Comment:
Since, this is a npm library, so for using, it should be imported something like this
`const DateValidatator = require('simple-date-validator);`

Reply:
Perfect, doing it now.